### PR TITLE
[NEW UI] Update requirements

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Update requirements
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | @PrestaShopCorp
| How to test?      | Test if the requirements section in Storybook is displayed correctly.

### CODE REVIEW

**Review by Quetzacoalt91 — APPROVED** [review]

File: views/templates/components/check-requirements.html.twig [hunk 1]
```diff
@@ -188,12 +188,12 @@
         {% endif %}
 
         {# Check for PHP compatibility warning #}
-        {% if phpRequirementsState and phpRequirementsState != constant('PrestaShop\\Module\\AutoUpgrade\\UpgradeSelfCheck::PHP_REQUIREMENTS_VALID') %}
-          <div class="check-requirements__requirement check-requirements__requirement--warning">
-            {% if phpRequirementsState != constant('PrestaShop\\Module\\AutoUpgrade\\UpgradeSelfCheck::PHP_REQUIREMENTS_INVALID') %}
+        {% if phpRequirementsState != constant('PrestaShop\\Module\\AutoUpgrade\\Services\\PhpVersionResolverService::COMPATIBILITY_VALID') %}
+          {% if phpRequirementsState != constant('PrestaShop\\Module\\AutoUpgrade\\Services\\PhpVersionResolverService::COMPATIBILITY_INVALID') %}
+            <div class="check-requirements__requirement check-requirements__requirement--warning">
```
Comment (Quetzacoalt91) [inline views/templates/components/check-requirements.html.twig]: To make the check simpler

```suggestion
        {% if phpRequirementsState == constant('PrestaShop\\Module\\AutoUpgrade\\Services\\PhpVersionResolverService::COMPATIBILITY_UNKNOWN') %}
          <div class="check-requirements__requirement check-requirements__requirement--warning">
```

File: views/templates/components/check-requirements.html.twig [hunk 2]
```diff
@@ -188,12 +188,12 @@
...
               {{ 'We were unable to check your PHP compatibility with the destination PrestaShop version.'|trans({}) }}
-            {% endif %}
-          </div>
+            </div>
+          {% endif %}
```
Comment (Quetzacoalt91) [inline views/templates/components/check-requirements.html.twig]:
```suggestion
          </div>
```

File: views/templates/components/check-requirements.html.twig [hunk 3]
```diff
@@ -23,14 +23,14 @@
       <div class="check-requirements__list">
         {# ERROR #}
         {# Check for PHP compatibility error #}
-        {% if phpRequirementsState and phpRequirementsState != constant('PrestaShop\\Module\\AutoUpgrade\\UpgradeSelfCheck::PHP_REQUIREMENTS_VALID') %}
-          {% if phpRequirementsState == constant('PrestaShop\\Module\\AutoUpgrade\\UpgradeSelfCheck::PHP_REQUIREMENTS_INVALID') %}
+        {% if phpRequirementsState != constant('PrestaShop\\Module\\AutoUpgrade\\Services\\PhpVersionResolverService::COMPATIBILITY_VALID') %}
+          {% if phpRequirementsState == constant('PrestaShop\\Module\\AutoUpgrade\\Services\\PhpVersionResolverService::COMPATIBILITY_INVALID') %}
```
Comment (Quetzacoalt91) [inline views/templates/components/check-requirements.html.twig]:
```suggestion
        {% if phpRequirementsState == constant('PrestaShop\\Module\\AutoUpgrade\\Services\\PhpVersionResolverService::COMPATIBILITY_INVALID') %}
```

File: views/templates/components/check-requirements.html.twig [hunk 4]
```diff
@@ -23,14 +23,14 @@
...
-                '%max_version%': phpCompatibilityRange['php_max_version'],
-                '%current_version%': phpCompatibilityRange['php_current_version']
-              }) }}
+              {{ 'Your current PHP version isn\'t compatible with your PrestaShop version. (Expected: %s - %s | Current: %s)'|trans([
+                phpCompatibilityRange['php_min_version'],
+                phpCompatibilityRange['php_max_version'],
+                phpCompatibilityRange['php_current_version'],
+              ]) }}
             </div>
           {% endif %}
```
Comment (Quetzacoalt91) [inline views/templates/components/check-requirements.html.twig]:
```suggestion
```

**Review by Quetzacoalt91 — DISMISSED** [review]
Actually, we can make the code easier to read.

**Review by ga-devfront — COMMENTED** [review]
I think it needs to be discussed, but in my opinion the list of requirements will not be hard-coded in the twig.

**Review by M0rgan01 — APPROVED** [review]

**Pull request merged**